pkg/config/config.go

```diff
@@ -96,6 +96,27 @@ func Empty() *Config {
 	return New(Opts{})
 }
 
+// Keys returns the possible config keys.
+func Keys() map[string]string {
+	return map[string]string{
+		"core.dcos_acs_token":   "the DC/OS authentication token",
```

Comment:
Ideally this would use the constants strings, to reduce the chance of having typos in the map keys.